src/formDesigner/components/ReportCard/ReportCardList.jsx

```
@@ -4,11 +4,12 @@ import { useNavigate } from "react-router-dom";
 import { useSelector } from "react-redux";
 import { Privilege } from "openchs-models";
 import { get } from "lodash";
+import {useNavigate} from "react-router-dom";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useNavigate'. Consider to delete it or rename it.

'useNavigate' is defined here:


(lint/suspicious/noRedeclare)